Makefile

```diff
@@ -41,6 +41,9 @@ publish-local:
     export IS_LOCAL_DEVELOPMENT=true; ./gradlew :services:uploadArchives ; \
     export IS_LOCAL_DEVELOPMENT=true; ./gradlew :services-turf:uploadArchives ; \
 
+graphs:
+	./gradlew generateDependencyGraphMapboxLibraries
```

Comment:
I'd include explicitly each module in which we're generating/including the dependency-graph plugin. It's clearer IMO.

```
graphs:
	./gradlew :services-core:generateDependencyGraphMapboxLibraries
	./gradlew :services-geojson:generateDependencyGraphMapboxLibraries
	./gradlew :services:generateDependencyGraphMapboxLibraries
	./gradlew :services-turf:generateDependencyGraphMapboxLibraries
```

Reply:
@Guardiola31337 I removed 
```apply from: "${rootDir}/gradle/dependencies-graph.gradle"```
from the top level `build.gradle` and now I do not have top level graph. 
Probably I should add one for samples, otherwise the graphs are not very meaningful.

Reply:
@osana 
> I removed
apply from: `"${rootDir}/gradle/dependencies-graph.gradle"`
from the top level `build.gradle` and now I do not have top level graph.

Yeah, that's correct. We're only interested in the library graphs (for now). The idea is to know what Mapbox dependencies include each library module. Samples wouldn't give us any extra value. Does that make sense?

In any case, we were getting an incorrect graph as commented in https://github.com/mapbox/mapbox-java/pull/791#discussion_r182170428 so another good reason to not include it.